### DISCUSSION TITLE
Make Docs `Rule::exists` like `Rule::in`

### DISCUSSION THREAD

**Discussion by baralogi** [dedoc/scramble#646] [Ideas]
what if we create a feature to display the allowed entries as generated by Rule::in on Rule::exists, because I think we need that too? Thanks

<img width="513" alt="Screenshot 2024-11-30 at 15 21 08" src="https://github.com/user-attachments/assets/ea563cd1-0a4a-4f14-962b-d391f4503537">

**Comment by romalytvynenko** [comment]
@baralogi `Rule::exists` can be used (usually is used?) to check for record existence in DB. If we try to eval it, this may result in hundreds of options. 

I'm aware there is cases where it may make sense, but I'm sure this is not an option to be a default.

**Reply by baralogi** [reply]
Sometimes it makes sense if there are only a few options, for example order status data which only has a few.  I think there should be a flagging to determine whether we want to display it in scramble or not. 

Big Thanks! 💥 💥 💥

**Reply by romalytvynenko** [reply]
@baralogi as a kind of workaround, you can use something like this:
```php
Rule::in(OrderStatus::query()->pluck('id'));
```
